metricbeat/default.toml

```diff
@@ -0,0 +1,57 @@
+[metricbeat_yaml]
```

Comment:
Is it possible to remove the `metricbeat_yaml` prefix on this? It looks like everything is underneath that key, but want to confirm that I'm not missing something.

Reply:
I think the purpose for this is to allow the plan to grow to support additional configuration files in the future as necessary. It's probably not safe to assume that the default.toml is only for this one configuration file.

Reply:
I'm for removing the `metricbeat_yaml` part of the TOML config also, keep it simple and in line with [RFC9](https://github.com/habitat-sh/core-plans-rfcs/blob/master/_RFCs/0009-plan-configuration-complexity.md)

Reply:
I exhaustively read through the whole metricbeat configuration documentation, and it appears that you can configure ALL of metricbeat via the `metricbeat.yml` file, so I agree.

If someone wants to use separate configuration files for each module, I'd recommend they make a wrapper package.